arches/app/templates/views/components/search/advanced-search.htm

```diff
@@ -8,7 +8,7 @@
     <div class="facets-container" style="">
         <div class="list-group search-facets">
             <div class="search-facet-item header">
-                <h4 class="search-facet-item-heading">{% trans "Search Facets" %}</h4>
+                <h4 class=" ">{% trans "Search Facets" %}</h4>
```

Comment:
Why add the search-facet-item-heading here?  This doesn't seem like it should be clickable.